src/test/e2e/Helpers/Selectors.ts

```diff
@@ -16,6 +16,17 @@ module.exports = {
   SearchOptionsTitle: 'h1.govuk-fieldset__heading',
   RadioButton: 'div.govuk-radios__item',
 
+
+  // Search page selectors
+  HaveNameRadio: '#find-choice',
+  DontHaveNameRadio: '#find-choice-2',
+
+  // Sign-in routing
+  Radio1: '#sign-in',
+  Radio2: '#sign-in-2',
+  Radio3: '#sign-in-3',
+  Radio4: '#sign-in-4',
```

Comment:
`SignInRadio...` would be a suitable prefix as the comments above dont apply to referencing the values, i.e. when you call it youll be saying `helpers.Radio1` which isnt as readable as we would like it

Reply:
refactored